src/macros.jl

```diff
@@ -2737,7 +2737,11 @@ macro variable(args...)
                 )
             end
         end
-        buildcall = :(build_variable($_error, $scalar_variables, $set))
+        buildcall = :(build_variable(
```

Comment:
For the constraints, we call `model_convert` on the output of `build_variable`, would is it different here ?

Reply:
Updated